MODULE.bazel

```diff
@@ -7,7 +7,8 @@ module(
 )
 
 bazel_dep(name = "aspect_bazel_lib", version = "1.32.1")
-bazel_dep(name = "aspect_rules_js", version = "1.29.2")
+bazel_dep(name = "aspect_rules_js", version = "1.31.0")
```

Comment:
Minimum rules_js version for Bazel 7 support https://pro.aspect.build/guides/bzlmod.pro/#bazel-7-minimum-versions